operator-pipeline-images/operatorcert/community_prow/start_community_prow.py

```diff
@@ -0,0 +1,11 @@
+"""
+Entrypoint module for community-pipelines prow tests
+"""
+
+
+def initiate_community_prow(*args, **kwargs):
```

Comment:
I would rather see this named as `main` (also file) invoked with `if __name__ == "__main__": main()`. Function does not have t o return anything. Also I would like to see here some log message proving that this is really triggered from community-operators-pipeline-preprod repo. After this, this have to be merged (also change on community side and tested, if log message is correctly displayed in prow logs. I appreciate also  @mporrato opinion about delegating this into separate repository grouping here all prow-side code logic.

Reply:
changed the naming and included logging